src/hooks/useProjects.ts

```diff
@@ -16,11 +14,6 @@ import { parseDBProject, parseDBProjectJson } from 'utils/sgDbProjects'
 
 const DEFAULT_STALE_TIME = 60 * 1000 // 60 seconds
 
-// Filter applied to all DB project query results
-// TODO this could be applied at the /api/projects level, but would take some more finessing. we're working quick here
```

Comment:
Frens, always do the finessing. Especially when it's actually not a lot of finessing

_Narrator: This was not a lot of finessing_

Reply:
Prime finesse right here.